java/gradle.java/log

```diff
@@ -0,0 +1,297 @@
+Buildfile: /space/src/vscode/netbeans/java/gradle.java/build.xml
```

Comment:
Unnecessary.

Reply:
Removed / recommitted in f9a8ede so it does not appear in the history.